src/messaging/ReliableMessageMgr.cpp

```diff
@@ -101,6 +101,29 @@ void ReliableMessageMgr::TicklessDebugDumpRetransTable(const char * log)
 #endif
 }
 
+#if CHIP_CONFIG_MRP_ANALYTICS_ENABLED
+void ReliableMessageMgr::NotifyMessageSendAnalytics(const RetransTableEntry & entry, const SessionHandle & sessionHandle,
+                                                    const ReliableMessageAnalyticsDelegate::EventType & eventType)
+{
+    if (!mAnalyticsDelegate)
+    {
+        return;
+    }
+    uint32_t messageCounter = entry.retainedBuf.GetMessageCounter();
+    auto fabricIndex        = sessionHandle->GetFabricIndex();
```

Comment:
Again for MRP for CASE messages: will this be the fabric index of the fabric the handshake is on?  Or (as I expect based on code inspection) will this be kUndefinedFabricIndex?  API needs to document, and then we should figure out what behavior would make sense here and maybe make it happen.

Reply:
Let me know if updated documentation looks good to you